testsuite/scripts/tutorials/test_11-ferrofluid_1.py

```diff
@@ -22,7 +22,7 @@
 
 tutorial, skipIfMissingFeatures = importlib_wrapper.configure_and_import(
     "@TUTORIALS_DIR@/11-ferrofluid/11-ferrofluid_part1.py",
-    equil_steps=200, equil_rounds=10)
+    equil_steps=200, equil_rounds=10, N=1200)
```

Comment:
If you changed `N` to `N_PART` in ff tutorial 1, you will also have to change that, here...